packages/block-library/src/video/save.js

```diff
@@ -37,6 +41,10 @@ export default function save( { attributes } ) {
 					preload={ preload !== 'metadata' ? preload : undefined }
 					src={ src }
 					playsInline={ playsInline }
+					className={ borderProps.className }
+					style={ {
+						...borderProps.style,
+					} }
```

Comment:
```suggestion
					style={ borderProps.style }
```

With only border styles being added here they don't need to be deconstructed.